src/sass/font.scss

```diff
@@ -47,3 +47,5 @@
 @import '~@fortawesome/fontawesome-free/scss/fontawesome.scss';
 @import '~@fortawesome/fontawesome-free/scss/regular.scss';
 @import '~@fortawesome/fontawesome-free/scss/solid.scss';
+@import '~gmf/sass/vars.scss';
+@import '~ngeo/sass/font.scss';
```

Comment:
Why this?

Reply:
It was in `contribs/gmf/src/sass/font.scss`

Reply:
But I did not checked what I reported ... :-)

Reply:
I think that better if we have two different files, the CSS generation is not strong then we wanted to avoid breaking it ...

Reply:
Ok, I did a new file called `gmf-font.scss` that contains the old file from contrib, and fixed the import that used this.